### ISSUE TITLE
Error using ANOVA with 2^3 factorial design with repetitions

### ISSUE THREAD

**Issue by vabatista** [raphaelvallat/pingouin#353]
I'm getting error with this code below:

```
y_measures = np.array([28,25,27,18,19,23,36,32,32,31,30,29,28,25,22,18,19,23,12,32,40,31,30,29])
factors = ['A' ,'B', 'C']
levels_list = [['low','high'],['low','high'],['low','high']]
replicates = 3

def generate_dataframe(measures, factors, levels_list, replicates):
    
    lines = []
    for factor_combination in product(*levels_list):
        line = {}
        for idx, factor in enumerate(factors):
            line[factor] = factor_combination[idx]
        for k in range(replicates):
            lines.append(line)
    df = pd.DataFrame(lines,columns=factors)
    df['y'] = measures
    return df

df = generate_dataframe(y_measures, factors, levels_list, replicates)

model1 = pg.anova(dv='y', between=factors, data=df, detailed=True)
```

Error:

```
PatsyError: Error evaluating factor: TypeError: 'Series' object is not callable 
    Q('y') ~ C(Q('A'), Sum) * C(Q('B'), Sum) * C(Q('C'), Sum)
```

Any ideas?

**Comment by remrama** [comment]
Yes, this is a known issue and has been fixed on PR #351 for the _next_ version of pingouin. For now, you can avoid the error by just renaming your column `"C"` to almost anything else.

**Comment by vabatista** [comment]
Thank you!
It also worked renaming to 'FactorC' ...
